src/controls/makeSubsetterControl.js

```diff
@@ -73,15 +73,16 @@ export default function makeSubsetterControl(control, control_wrap) {
                 col: control.value_col,
                 val: values,
                 choices: option_data,
-                loose: control.loose
+                loose: control.loose,
+                draw: true
             };
             targets.forEach(e => {
                 setSubsetter(e, new_filter);
                 //call callback function if provided
                 if (control.callback) {
                     control.callback();
                 }
-                e.draw();
+                if (control.draw) e.draw();
```

Comment:
So the use case is only for subsetter controls?

Reply:
No, it was for all controls - might've missed a spot though. I thought the update in `scr/controls/changeOption.js` covered the other use case - no?

Reply:
quite right, quite right